mcs/class/System.Private.CoreLib/System/AppDomainSetup.cs

```diff
@@ -0,0 +1,38 @@
+using System.Runtime.InteropServices;
+
+namespace System
+{
+	[StructLayout (LayoutKind.Sequential)]
+	public sealed class AppDomainSetup
```

Comment:
This type does really not exist in Core in this shape, we should not add it at all

Reply:
until domains are dead in the runtime, this can't go away.  Right now we create a root AppDomain and its AppDomainSetup